Tweaks to the script to replace AppVeyor with the new GitHub builds.

essasetic opened the pull request:
As the name implies, it makes slight tweaks to the script in-order to accommodate the sudden switch for Ryujinx using github.

This is based on the testing branch so if you want me to merge it there or base it on the master branch or if you're perfectly fine with merging it anyway let me know.

The script should work as it did before.

Closes #4

essasetic committed:
WIP modifications

Begins the transfer from AppVeyor to Github

essasetic committed:
Update README.md

essasetic committed:
It works!!

essasetic committed:
Remove hardcoding of the 1.1.4 version

Makes it so it can be upgraded when a new version comes out.

MGThePro commented on README.md:
```diff
@@ -9,7 +8,7 @@ The purpose of the installer is to place Ryujinx inside ~/.local/share/Ryujinx,
 Usage:
 Open a terminal, paste <br>
-`bash -c "$(curl -s https://raw.githubusercontent.com/edisionnano/Pine-jinx/main/pinejinx.sh)"` <br>
+`bash -c "$(curl -s https://raw.githubusercontent.com/essasetic/Pine-jinx/main/pinejinx.sh)"` <br>
```
Might want to change this back to the original repo for the PR

essasetic replied:
Ah yeah, oversight on my end.

MGThePro commented on pinejinx.sh:
```diff
@@ -98,41 +113,46 @@ install () {
 	rm -rf /home/${USER}/.local/share/mime/packages/Ryujinx.xml
 	rm -rf /home/${USER}/.local/share/applications/Ryujinx.desktop
 	rm -rf /home/${USER}/.local/share/icons/Ryujinx.png
+	rm -rf /home/${USER}/.config/gamemode.ini
```
I know this change is from the testing branch, but I don't think it's safe to assume that deleting the gamemode config is fine on every system. Same goes for overwriting it in line 75. Ideally there would be a way to pick a custom config just for ryujinx via env variable or parameter, but I'm not sure if that's an option

essasetic replied:
Yeah I agree, but I'm simply hotfixing the script to make it work with github and don't really have any experience with shell. So I'll have to bring that one up to the maintainer.

MGThePro replied:
removing line 125 and 75 should be sufficient, I don't see the script touching the gamemode.ini anywhere else

MGThePro commented on README.md:
```diff
@@ -1,5 +1,4 @@
-**WARNING:If you are on Ubuntu or an Ubuntu-based distribution you'll need to do `sudo apt install libx11-dev` before using Pine-jinx!**
-
+**WARNING:If you are on Ubuntu or an Ubuntu-based distribution you'll need to do `sudo apt install libx11-dev` before using Pine-jinx! Also please note that this fork is a heavy WIP and only grabs 1.1.4 as of now. This only affects the master build, not the LDN build (as its hosted on patreon)**
```
This also seems outdated

essasetic replied:
Yeah I'll change that back. I only made it when I began work on it last night.

essasetic committed:
Warning no longer necessary.

essasetic committed:
Change URL back to original.

edisionnano commented:
I'm not yet ready to push testing to main. First of all because adding chmod on the desktop file breaks GNOME.

essasetic committed:
Update Ryujinx.desktop

essasetic committed:
Create gamemode.ini

essasetic commented:
> I'm not yet ready to push testing to main. First of all because adding chmod on the desktop file breaks GNOME.

Removed that from the desktop file, also added everything else from the testing branch as I somehow forgot to do so beforehand

essasetic commented:
![image](https://user-images.githubusercontent.com/34970796/150682834-530ecf53-26f1-4864-a360-b59fb3e1ede6.png)

Don't merge yet. This is a big issue.

essasetic committed:
Remove tar.gz exclusion

essasetic committed:
Fix ryujinx.desktop

essasetic commented:
> ![image](https://user-images.githubusercontent.com/34970796/150682834-530ecf53-26f1-4864-a360-b59fb3e1ede6.png)
> 
> Don't merge yet. This is a big issue.

Ok so it seems manually removing the /tmp/pineapple file solves this but I'm not sure why the program isn't doing this automatically. It could be solved now maybe?

EDIT: Applied the hardcoded 1.1.4 version, installed it, then undid the changes and got 1.1.5 installed with no tar errors. Think we're good to go!

essasetic committed:
Simplify removing /tmp/pineapple directory

essasetic committed:
Merge branch 'main' into patch-1

essasetic committed:
Merge pull request #1 from Essasetic/patch-1

Simplify removing /tmp/pineapple directory

essasetic committed:
Changing paths from "testing" to "main"

essasetic committed:
Add gamemode instructions
